DemoSwiftApp/Samples/SamplesForAPI.swift

```diff
@@ -320,7 +321,7 @@ class SamplesForAPI {
             print("[AudienceSegments] decision: \(decision)")
         }
         
-        if #available(iOS 13, *) {
+        if #available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *) {
             Task { [user] in
```

Comment:
Do we need this explicit capturing [user] here?

Reply:
Yah, as we call from the static method, that's why we needed to capture it. Otherwise, the compiler throws errors. Normally we don't need to capture it.